analysis.go

```diff
@@ -50,60 +49,51 @@ type DeploymentConfigs struct {
 	} `json:"items"`
 }
 
-type CheckTask func(string, io.Writer) (Result, error)
+type CheckTask func(string, string) (Result, error)
+
+func GetAnalysisTasks(tasks chan<- Task, basepath string, projects []string, results chan<- CheckResults) {
+	// Platform-wide analysis goes here
+
+	// project specific analysis in here
+	for _, p := range projects {
```

Comment:
There's not much here at the moment, but as more analysis tasks are added this function will grow.